hi-ml/testhiml/testhiml/test_attentionlayers.py

```diff
@@ -59,8 +61,18 @@ def test_max_pooling(dim_in: int, batch_size: int,) -> None:
 @pytest.mark.parametrize("num_heads", [1, 2])
 @pytest.mark.parametrize("dim_in", [4, 8])   # dim_in % num_heads must be 0
 @pytest.mark.parametrize("batch_size", [1, 7])
-def test_transformer_pooling(num_layers: int, num_heads: int, dim_in: int, batch_size: int) -> None:
+@pytest.mark.parametrize("transformer_dropout", [0.0, 0.3])
+@pytest.mark.parametrize("dim_hid", [1, 4])
+def test_transformer_pooling(num_layers: int, num_heads: int, dim_in: int,
+                             batch_size: int, dim_hid: int, transformer_dropout: float) -> None:
     transformer_pooling = TransformerPooling(num_layers=num_layers,
                                              num_heads=num_heads,
-                                             dim_representation=dim_in).eval()
+                                             dim_representation=dim_in,
+                                             transformer_dropout=transformer_dropout).eval()
     _test_attention_layer(transformer_pooling, dim_in=dim_in, dim_att=1, batch_size=batch_size)
+    transformer_pooling_benchmark = TransformerPoolingBenchmark(num_layers=num_layers,
```

Comment:
I'd suggest testing this component in a separate test function or test case, so we can catch errors separately for both. Also, they have different input arguments.

Reply:
Thanks, now separated the tests